libs/ui/src/lib/steps/steps/steps.component.html

```diff
@@ -41,6 +41,23 @@ <h1 *ngIf="title" class="cvi-steps__title" dataAttribute="steps_title">{{ title
           </cvi-ng-track>
         </button>
       </li>
+      <li *ngIf="urlStepTitle"
+          class="cvi-steps__list-item cvi-steps__display_desktop">
```

Comment:
```suggestion
          class="cvi-steps__list-item cvi-steps__list-item--is-url-step">
```
This combination of classes is invalid according to the BEM methodology used in CVI – two "elements" with `__` can't be located at the same DOM node. An easy solution is to use a "modifier" class (the one that modifies `cvi-steps__list-item` with `--` suffix) instead. OR you can simply use `:last-child` selector with `cvi-steps__list-item`.

Reply:
Added modifier